docs/docsite/rst/community/steering/community_steering_committee.rst

```diff
@@ -46,6 +46,8 @@ The following table lists the current Steering Committee members. See :ref:`stee
   +------------------+---------------+-------------+
   | Brian Scholer    | briantist     | 2022        |
   +------------------+---------------+-------------+
+  | Core Team repr.  | See Forum     | 2024        |
```

Comment:
If the Community Topic in the forum exists, I think we should have a link to it here.

Reply:
It doesn't exist yet but how about just adding a link afterwards w/o a vote? We first need to vote on this PR imo as the core-team representative announcement topic depends on approval of this PR by SC,
If you want to create it now it wouldn't hurt either, so please go ahead and paste the link here.

Reply:
Sorry, I think I've put this badly. I should have said _when_ the Community Topic in the forum exists.

So vote on this, merge your PR (if the vote is positive) and add the link afterwards.